openghg/util/_met.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+
+
+def _get_site_pressure(inlet_heights: list, site_height: float) -> list[float]:
+    """Calculate the pressure levels required
```

Comment:
Just a quick note on the output units here (e.g. Pa? hPa?) would be helpful here.